src/frontend/src/pages/FinancePage/ReimbursementRequestForm/ReimbursementFormView.tsx

```diff
@@ -187,19 +193,28 @@ const ReimbursementRequestFormView: React.FC<ReimbursementRequestFormViewProps>
                 <Controller
                   name="expenseTypeId"
                   control={control}
-                  render={({ field: { onChange, value } }) => (
-                    <Select
-                      onChange={(newValue) => onChange(newValue.target.value)}
-                      value={value}
-                      error={!!errors.expenseTypeId}
-                    >
-                      {allExpenseTypes.map((expenseType) => (
-                        <MenuItem key={expenseType.expenseTypeId} value={expenseType.expenseTypeId}>
-                          {expenseType.name}
-                        </MenuItem>
-                      ))}
-                    </Select>
-                  )}
+                  render={({ field: { onChange, value } }) => {
+                    return (
+                      <NERAutocomplete
+                        options={allExpenseTypes.map(expenseTypesToAutocomplete)}
```

Comment:
because you call `allExpenseTypes.map(expenseTypesToAutocomplete)`, can you make it a variable and then call it in both places you use it?